src/Components/test/E2ETest/ServerRenderingTests/StreamingRenderingTest.cs

```diff
@@ -268,4 +268,11 @@ public async void StopsProcessingStreamingOutputFromPreviousRequestAfterEnhanced
         // Tidy up
         await new HttpClient().GetAsync(endResponseUrl);
     }
+
+    [Fact]
+    public void CanStreamDirectlyIntoSectionContentConnectedToNonStreamingOutlet()
+    {
+        Navigate($"{ServerPathBase}/streaming-with-sections");
+        Browser.Equal("This is some streaming content", () => Browser.Exists(By.Id("streaming-message")).Text);
```

Comment:
Is my understanding correct here that at no point we want to see "Is streaming..."?

Reply:
That does happen, but observing that in the test isn't necessary to validate that the bug is fixed and the feature works. Before the bug was fixed, you'd never see the final "This is some streaming content" message at all, and now you do.